siliconcompiler/schema/schema_obj.py

```diff
@@ -134,13 +134,21 @@ def _read_manifest(filepath):
 
         return localcfg
 
-    ###########################################################################
     def get(self, *keypath, field='value', job=None, step=None, index=None):
         """
         Returns a schema parameter field.
 
         See :meth:`~siliconcompiler.core.Chip.get` for detailed documentation.
         """
+        # Prevent accidental modifications of the schema content by not passing a reference
+        result = self.__get(*keypath, field=field, job=job, step=step, index=index)
+        try:
+            return result.copy()
+        except AttributeError:
+            return result
```

Comment:
can we use:
```
import copy
...
return copy.copy(result)
```

instead? seems cleaner than try/except

Reply:
Sure that works too.

Reply:
Thanks for the suggestion I was looking for something like that.
Done now